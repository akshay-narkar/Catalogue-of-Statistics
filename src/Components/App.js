import '../App.css';
import Hompage from '../Containers/homepage';

function App() {
  return (
    <>
      <Hompage />
    </>
  );
}

export default App;
