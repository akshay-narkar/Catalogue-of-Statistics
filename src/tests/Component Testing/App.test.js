/*eslint-disable*/

import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import React from 'react';
import App from '../../Components/App';
import store from './store';

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<BrowserRouter><Provider store={store}><App /></Provider></BrowserRouter>).toJSON();

  expect(elem).toMatchSnapshot();
});
