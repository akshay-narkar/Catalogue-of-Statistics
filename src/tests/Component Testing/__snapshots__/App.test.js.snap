// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
<h1>
  Oops! An error occured in rendering!
</h1>
`;
